pkg/project/auth/cache.go

```diff
@@ -242,6 +281,36 @@ func (ac *AuthorizationCache) synchronizePolicyBindings(userSubjectRecordStore c
 	}
 }
 
+// synchronizePolicies synchronizes access over each policy
+func (ac *AuthorizationCache) synchronizeClusterPolicies(userSubjectRecordStore cache.Store, groupSubjectRecordStore cache.Store, reviewRecordStore cache.Store) {
+	items := ac.clusterPolicyIndexer.List()
+	for i := range items {
+		policy := items[i].(*authorizationapi.ClusterPolicy)
+		reviewRequest := &reviewRequest{
+			namespace:                  policy.Namespace,
+			policyUIDToResourceVersion: map[types.UID]string{policy.UID: policy.ResourceVersion},
+		}
+		if err := ac.syncHandler(reviewRequest, userSubjectRecordStore, groupSubjectRecordStore, reviewRecordStore); err != nil {
+			util.HandleError(fmt.Errorf("error synchronizing: %v", err))
+		}
+	}
+}
+
+// synchronizePolicyBindings synchronizes access over each policy binding
+func (ac *AuthorizationCache) synchronizeClusterPolicyBindings(userSubjectRecordStore cache.Store, groupSubjectRecordStore cache.Store, reviewRecordStore cache.Store) {
```

Comment:
Get rid of this as well.  It has no purpose.  We get no benefit finding out if the user has access to the empty string namespace.